### DISCUSSION TITLE
Format Selection Does not work for Playlists

### DISCUSSION THREAD

**Discussion by Mine181** [JunkFood02/Seal#468] [Q&A] [answered]
I'm so happy to see format selection where I can select exact video qualities, containers, audio inclusion, or audio only qualities and formats! But it seems like it only works for individual videos and not playlists, is this a bug or was this not considered?

**Comment by JunkFood02** [comment, marked as answer]
Currently it's not considered to be implemented, videos from playlists will be downloaded in suggested formats. It's complicated to merge two features together and we're only able to obtain the format info of individual videos, not all of them of the whole playlist.

**Reply by jengablockmoneyspread** [reply]
I been looking to see this exact thing if there is quality selection for playlists. Has this been added? I feel like splitting the playlist into its separate video links then allowing for selection of video quality for each video to be a rudementary way of doing this so I'd just like to know because I have a low monthly data allowance thus I cant experiment much.
